lldb/source/Plugins/SymbolFile/DWARF/DebugNamesDWARFIndex.h

```diff
@@ -65,6 +65,10 @@ class DebugNamesDWARFIndex : public DWARFIndex {
   void GetFunctions(const RegularExpression &regex,
                     llvm::function_ref<bool(DWARFDIE die)> callback) override;
 
+  StatsDuration::Duration GetIndexTime() override {
```

Comment:
Given the case where we have full debug names, would we want to return the `m_index_time` instead of `m_fallback.GetIndexTime()`? I don't have context to know if debug names itself ever has an index time, or that it will only ever be `> 0` when use the manual indexer.